pom.xml

```diff
@@ -50,10 +50,10 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <!--<jenkins.version>2.46.3</jenkins.version>-->
-        <java.level>6</java.level>
-        <!--<java.level>7</java.level>-->
+        <jenkins.version>2.60.2</jenkins.version>
```

Comment:
Cool, I don't think the code depends on anything new, but for my testing wanted to align it with our Jenkins version.